integrations/google-analytics-4/test/index.js

```diff
@@ -145,20 +145,21 @@ describe('Google Analytics 4', function () {
             analytics.equal(window.ga4DataLayer[2][2]['cookie_expires'], 21)
 
             // cookie_flags uses the `set` command
-            analytics.equal(window.ga4DataLayer[3][0]['cookie_flags'], 'SameSite=None;Secure')
+            analytics.equal(window.ga4DataLayer[3][0], 'set')
```

Comment:
Should we use a `deepEqual` here like the other updated tests!

Reply:
Yeah, that may be cleaner. I'll change before I merge!

Reply:
Nice! Definitely a nit - but it does look nice this way! Thank you!